validated_arch_1/stage5/openstackdataplanenodeset.yaml

```diff
@@ -53,8 +53,13 @@ spec:
       ansiblePort: 22
       ansibleUser: cloud-admin
       ansibleVars:
+        # CHANGEME
         edpm_chrony_ntp_servers:
           - clock.redhat.com
+        # CHANGEME -- see https://access.redhat.com/solutions/253273
+        # edpm_bootstrap_command: |
+        #       subscription-manager register --username <subscription_manager_username> --password <subscription_manager_password>
+        #       podman login -u <registry_username> -p <registry_password> registry.redhat.io
```

Comment:
Would you mind adding something for these to the `CHANGEME` list at the top as well, like we've done for other `CHANGEME`s?  I think it's good enough to just have it in this file and not to need repeating in the stage6 version, as that version refers back to any customizations made to this version.

Reply:
There is already a service created in ci-framework and is used in downstream jobs. https://github.com/openstack-k8s-operators/ci-framework/commit/a793f8f18ab8d3b1f8da3e164992f6a26f4c467e